bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/BitcoindRpcClient.scala

```diff
@@ -275,6 +275,13 @@ class BitcoindRpcClient(override val instance: BitcoindInstance)(implicit
     }
   }
 
+  def generate(numBlocks: Int): Future[Vector[DoubleSha256DigestBE]] = {
```

Comment:
Might want to provide an optional address here if you want to fund a wallet

```scala
def generateToAddress(numBlocks: Int, addressOpt: Option[BitcoinAddress] = None): Futur[Vector[DoubleSha256DigestBE]]
```

Reply:
Eh, actually if you keep the old `generateToAddress` around maybe it isn't worth it 🤷‍♂️

Reply:
Yeah that's what generate to address is for